block_snippets/response.py

```diff
@@ -16,19 +18,24 @@ def __init__(self, *args, **kwargs):
 
     def render_snippet(self, template, context, snippet_name):
         snippet_template = context.render_context.get('snippets', {}).get(snippet_name)
-        if django.get_version() >= '1.8':
+        if StrictVersion(django.get_version()) >= StrictVersion('1.8'):
             return snippet_template._rendered_context if snippet_template is not None else ''
         else:
             return snippet_template.render_content(context) if snippet_template is not None else ''
 
     @property
     def rendered_content(self):
-        if django.get_version() >= '1.8':
-            from django.template.context import make_context
-
+        if (StrictVersion(django.get_version()) < StrictVersion('1.8') or
+                StrictVersion(django.get_version()) >= StrictVersion('1.10')):
+            template = self.resolve_template(self.template_name)
+            context = self.resolve_context(self.context_data)
```

Comment:
Nebylo by lepší to udělat takto?
```python
if hasattr(self, 'resolve_template') and hasattr(self, '_resolve_context'):
    ...
else:
    ...
```
Je to kompatibilní a není to vázané na verze.